app/assets/stylesheets/components/_theme.scss

```diff
@@ -0,0 +1,71 @@
+$Theme-spacing-base: 18px;
+$Theme-spacing-small: 9px;
+
+$Theme-fontSize-base: 18px;
+$Theme-fontSize-large: 22px;
```

Comment:
There you're saying spacing-small is half of the base font-size. It's just a coincidence that the base font-size is the same as the base spacing. So the only coupling that could exist would be between the base font-size and the base spacing like:
```
$Theme-spacing-base: 1rem
$Theme-spacing-small: $Theme-spacing-base / 2
```